build.gradle

```diff
@@ -772,6 +775,37 @@ distributions {
     }
   }
 }
+
+ospackage {
+  packageName = distributions.main.baseName
```

Comment:
Add "-without-hadoop" suffix for "hadoop-provided" setting like being done in distTar/distZip